### PR TITLE
Update the Time rule to a complete example and extend the explanation of .cancel()

### PR DESCRIPTION
**Pull request by Trysupe**
Source https://community.openhab.org/t/difference-between-timer-null-and-timer-cancel/116363/2

 Signed-off-by: Felix Schneider <fs@felix-schneider.org>

### CODE REVIEW

File: addons/actions.md
Comment (Confectrician) [inline addons/actions.md:134]: I think you forgot your **heizdecke** one time. 😛

**Review by Confectrician — COMMENTED** [review]
Thanks Felix 👍

One small comment below.

I will fix the sign-off error before merging.
Also don't worry about the failing preview build. That is in our responibility.

**Comment by Trysupe** [comment]
> One small comment below.
> 
> I will fix the sign-off error before merging.
> Also don't worry about the failing preview build. That is in our responibility.

I know. I didn't find out on how to sign the commits in the web editor. I couldn't clone the repo and do it locally since my vscode has many extension that change all lines (removing trailing spaces, etc...)

I will fix the "Heizdecke" later this evening

**Comment by Confectrician** [comment]
>I didn't find out on how to sign the commits in the web editor.

The statement should be placed in the last line (description input) for each commit.



Reference:
https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330?u=confectrician

I have a [Beeftext-Combo](https://beeftext.org/) for this, when adding small changes in web directly.

**Comment by Trysupe** [comment]
> The statement should be placed in the last line (description input) for each commit.

Okay. Thanks. I'll try later when editing my original commit

**Comment by Trysupe** [comment]
I've removed the `heizdecke`. As far as I can see the signing didn't work. I added my signature in the last (and only) line of the description of the commit...

**Comment by Confectrician** [comment]
No problem, i will override that.

Some more information for explanation.
According to https://github.com/openhab/openhab-docs/pull/1482/checks there area two problems left.

- First commit is not signed off. (That was expected.)
- Second commit is signed off, but the DCO Check expected your github-noreply adress
  This could be caused, when your sign-off email is not connected to your GitHub-Account as far as i know.
  (Could even be possible that it has to be public visible, but i am not sure about that.)

So the next question would be: Do you use the sign-of mail in your github account and/or is it public visible?

Anyways for me that's enough, because we will have a proper sign-off statement in the merge commit. 🙂

**Review by Confectrician — APPROVED** [review]
Thanks 👍

**Pull request merged**
